packages/genesys-spark-components/src/components/stable/gux-icon/readme.md

```diff
@@ -10,7 +10,7 @@
o-disable" \| "deprecated/view-all" \| "deprecated/view-cells" \| "deprecated/view-grid" \| "deprecated/view-list" \| "deprecated/view-lists" \| "deprecated/view-roster" \| "deprecated/view-stacked" \| "deprecated/voicemail" \| "deprecated/volume-disable" \| "deprecated/volume-down" \| "deprecated/volume-none" \| "deprecated/volume-up" \| "deprecated/web" \| "deprecated/webcam" \| "deprecated/webcam-disable" \| "deprecated/wechat" \| "deprecated/whatsapp" \| "deprecated/window-actual-size" \| "deprecated/window-contract" \| "deprecated/window-drag-out" \| "deprecated/window-expand" \| "deprecated/window-fullscreen" \| "deprecated/window-popout" \| "deprecated/wordpress" \| "deprecated/youtube" \| "deprecated/zoom-in" \| "deprecated/zoom-out" \| "device-camera" \| "device-cellphone" \| "device-headphones" \| "device-laptop" \| "device-pc" \| "device-server" \| "device-tablet" \| "download" \| "edge" \| "edit" \| "email-notification-score" \| "expand" \| "export" \| "external-link" \| "fa/00-regular" \| "fa/00-solid" \| "fa/1-regular" \| "fa/address-book-regular" \| "fa/address-card-regular" \| "fa/alarm-clock-regular" \| "fa/alarm-snooze-regular" \| "fa/align-center-regular" \| "fa/align-justify-regular" \| "fa/align-left-regular" \| "fa/align-right-regular" \| "fa/anchor-regular" \| "fa/android" \| "fa/apple" \| "fa/arrow-down-left-and-arrow-up-right-to-center-regular" \| "fa/arrow-down-left-regular" \| "fa/arrow-down-regular" \| "fa/arrow-down-right-regular" \| "fa/arrow-down-to-line-regular" \| "fa/arrow-down-wide-short-regular" \| "fa/arrow-left-regular" \| "fa/arrow-left-to-line-regular" \| "fa/arrow-left-to-line-solid" \| "fa/arrow-right-arrow-left-regular" \| "fa/arrow-right-from-bracket-regular" \| "fa/arrow-right-regular" \| "fa/arrow-right-to-bracket-regular" \| "fa/arrow-right-to-line-regular" \| "fa/arrow-right-to-line-solid" \| "fa/arrow-rotate-left-regular" \| "fa/arrow-turn-down-right-regular" \| "fa/arrow-up-from-line-regular" \| "fa/arrow-up-left-regular" \| "fa/arrow-up-regular" \| "fa/arrow-up-right-and-arrow-down-left-from-center-regular" \| "fa/arrow-up-right-from-square-regular" \| "fa/arrow-up-right-regular" \| "fa/arrows-down-to-people-regular" \| "fa/arrows-maximize-regular" \| "fa/arrows-minimize-regular" \| "fa/arrows-rotate-regular" \| "fa/arrows-rotate-solid" \| "fa/arrows-to-circle-regular" \| "fa/arrows-up-down-regular" \| "fa/asterix-regular" \| "fa/at-regular" \| "fa/award-solid" \| "fa/backward-fast-regular" \| "fa/backward-regular" \| "fa/backward-step-regular" \| "fa/badge-check-solid" \| "fa/ban-outline" \| "fa/ban-regular" \| "fa/bars-regular" \| "fa/bell-on-regular" \| "fa/bell-regular" \| "fa/bell-slash-regular" \| "fa/binoculars-regular" \| "fa/block-brick-regular" \| "fa/bold-regular" \| "fa/bolt-regular" \| "fa/book-regular" \| "fa/bookmark-regular" \| "fa/border-all-regular" \| "fa/box-archive-regular" \| "fa/box-regular" \| "fa/brackets-square-regular" \| "fa/briefcase-regular" \| "fa/browser-regular" \
```

Comment:
With this many icons, maybe we should discuss whether the automatic script for the icon name attribute is worth it. Feels like it just makes the readme less readable